src/Compilers/VisualBasic/Portable/Binding/Binder_Invocation.vb

```diff
@@ -3106,7 +3120,7 @@ ProduceBoundNode:
 
                 ' For compatibility with the native compiler bad metadata constants should be treated as default(T).  This 
                 ' is a possible outcome of running an obfuscator over a valid DLL 
-                If defaultConstantValue.IsBad Then
+                If defaultConstantValue Is Nothing OrElse defaultConstantValue.IsBad Then
```

Comment:
Isn't this already under the If defaultConstantValue IsNot Nothing? If so, please remove.